### DISCUSSION TITLE
Decrease Width of Date Picker

### DISCUSSION THREAD

**Discussion by JayP718** [Budibase/budibase#8995] [Help] [answered]
![image](https://user-images.githubusercontent.com/42209052/206725688-a64d5953-cece-43c0-bcdf-f5e2e06c8896.png)

Hi,

Does anyone know how to decrease size of the date picker. 

I've tried to reduce the width but it only slightly changes the width of the date picker. I've looked at the adobe spectrum documentation but not having much luck there either. 

Much appreciated.

Best,

Jay

**Comment by andz-bb** [comment, marked as answer]
Hey @JayP718 

In this case I think the min-width is being enforced by the style on the `spectrum-InputGroup` class. I was able to override this by providing a custom style for that class inside an embed like so:

```
<style>
  .spectrum-InputGroup {
    min-width: 120px;
  }
</style>
```

Then when I also set the component's width to 120px it displays as that width as expected.

<img width="1285" alt="image" src="https://user-images.githubusercontent.com/110921612/207060521-ce21816b-0633-4ff2-aaa9-e658e5c3ba31.png">

You could also use the `spectrum-Datepicker` class to only apply it to Date Picker components, but it's pretty safe using the input group class as it's only setting the minimum width. Hope this helps!

**Reply by JayP718** [reply]
Thank you!!!!!!!

**Reply by andz-bb** [reply]
@JayP718 you're very welcome, glad I could help! 😎
